test/lib/server-http2.ts

```diff
@@ -360,11 +363,11 @@ export class ServerHttp2 extends TypedServer< Http2Server >
 }
 
 export async function makeServer( opts: ServerOptions = { } )
-: Promise< { server: Server; port: number | null; } >
+: Promise< { server: Server; port: number | null; receivedCookies: Array<string> } >
```

Comment:
[minor]: As i described before use constructors as types is not a good idea.

```suggestion
: Promise< { server: Server; port: number | null; receivedCookies: string[] } >
```